packages/types/src/admin-meta.ts

```diff
@@ -1,7 +1,34 @@
+import { GraphQLError } from 'graphql';
 import type { ReactElement } from 'react';
 import { GqlNames, JSONValue } from './utils';
 
-export type AdminConfig = { components?: { Logo?: (props: {}) => ReactElement } };
+export type NavigationProps = {
+  authenticatedItem: AuthenticatedItem;
+  lists: ListMeta[];
```

Comment:
What's the rationale behind providing these as props over users using `useKeystone`?

Reply:
useKeystone provides lists as a record, and visibleLists as an array of strings. 
The rationale is mostly DX, we already do the work to create an array of visible ListMeta items in the Navigation component, surfacing this to the user cuts out the boilerplate of 

```
Object.keys(lists).map(list => ...filter out non visible lists and create a new array of lists) 
```